en/flying/basic_flying.md

```diff
@@ -37,16 +37,14 @@ A VTOL vehicle can only arm in multicopter mode (by default - arming in fixed-wi
 
 ## Takeoff
 
-The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode](../flight_modes/takeoff.md).
+### Multicopter takeoff
+
+The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode (MC)](../flight_modes_mc/takeoff.md).

```

Comment:
I wouldn't call this the easiest way to takeoff with a MC. TBH I would completely remove the reference here (it's all about manual flying), or max have a reference further down that takeoff could also be automatic and then the user can take over.
Similar story for FW.